src/main/java/org/superbiz/moviefun/rest/MoviesResource.java

```diff
@@ -68,6 +69,14 @@
     @Claim("jti")
     private ClaimValue<String> jti;
 
+
+    @Inject
+    @DecryptedValue("creditCard")
+    private String creditCard;
+
+    @Inject
+    private Person person;
```

Comment:
I believe the username, email belongs to the `Person` .
Maybe remove these fields and just inject the person